linux/Makefile

```diff
@@ -0,0 +1,7 @@
+
+all:
+	go build -buildmode=plugin -o shim-linux-amd64.so
+
+install:
+	mkdir -p /var/lib/containerd/plugins
```

Comment:
`/usr/lib(64)?/containerd/plugins`?  Apache httpd does `/usr/lib(64)?/httpd/modules` and Firefox does `/usr/lib(64)?/mozilla/plugins`.